website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -86,53 +86,85 @@ HCP Terraform prioritizes and overwrites conflicting variables according to the
 
 For example, if you apply `A_Variable_Set` and `B_Variable_Set` to the same workspace, HCP Terraform will use any conflicting variables from `A_Variable_Set`. This is the case regardless of which variable set has been edited most recently. HCP Terraform only considers the lexical ordering of variable set names when determining precedence.
 
-### 8. Project-scoped variable sets
+Variables sets owned by projects take precedence over those owned by organizations.
+
+### 10. Workspace-scoped, organization-owned variable sets
+
+Variables in workspace-scoped variable sets are only applied to a subset of workspaces in an organization.
+
+When workspace-scoped variable sets have conflicting variables, HCP Terraform compares the variable set names and uses values from the variable set with lexical precedence. Terraform and HCP Terraform operate on UTF-8 strings, and HCP Terraform sorts variable set names based on the lexical order of Unicode code points.
+
+For example, if you apply `A_Variable_Set` and `B_Variable_Set` to the same workspace, HCP Terraform will use any conflicting variables from `A_Variable_Set`. This is the case regardless of which variable set has been edited most recently. HCP Terraform only considers the lexical ordering of variable set names when determining precedence.
+
+### 11. Project-scoped, project-owned variable sets
+
+Workspace-specific variables and workspace-scoped variable sets always take precedence over project-scoped variable sets that are applied to workspaces within a project.
+
+Variables in project-scoped variable sets are only applied to the workspaces within the specified projects.
+
+When project-scoped variable sets have conflicting variables, HCP Terraform compares the variable set names and uses values from the variable set with lexical precedence. Terraform and HCP Terraform operate on UTF-8 strings, and HCP Terraform sorts variable set names based the on lexical order of Unicode code points.
+
+For example, if you apply `A_Variable_Set` and `B_Variable_Set` to the same project, HCP Terraform uses any conflicting variables from `A_Variable_Set`. This is the case regardless of which variable set has been edited most recently. HCP Terraform only considers the lexical ordering of variable set names when determining precedence.
+
+Variables sets owned by projects take precedence over those owned by organizations.
+
+### 12. Project-scoped, organization-owned variable sets
 
-Workspace-specific variables and workspace-scoped variable sets always take precedence over project-scoped variable sets that are applied to workspaces within a project. 
+Workspace-specific variables and workspace-scoped variable sets always take precedence over project-scoped variable sets that are applied to workspaces within a project.
```

Comment:
Again, don't feel like we need this, and can you update this to mention organizations?